src/services/calendar/index.js

```diff
@@ -188,40 +200,51 @@ class Service {
 			},
 			body: convertEventToJsonApi(data),
 			json: true,
-			timeout: REQUEST_TIMEOUT,
+			timeout: Configuration.get('REQUEST_TIMEOUT_MILLIS'),
 		};
 
 		return request(options).then((events) => {
-			events = (events.data || []).map((event) => Object.assign(event, {
-				title: event.summary,
-				allDay: false, // TODO: find correct value
-				start: Date.parse(event.dtstart),
-				end: Date.parse(event.dtend),
-				url: '', // TODO: add x-sc-field
-			}));
+			events = (events.data || []).map((event) =>
+				Object.assign(event, {
```

Comment:
Codacy found an issue: [Expected no linebreak before this expression.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)